cfme/tests/automate/test_service_automate.py

```diff
@@ -139,3 +141,73 @@ def test_user_requester_for_lifecycle_provision(request, appliance, provider, se
         provision_request.wait_for_request(method='ui')
         request.addfinalizer(provision_request.remove_request)
         assert result.validate(wait="60s")
+
+
+@pytest.fixture(scope="module")
+def setup_dynamic_dialog(appliance, custom_instance):
+    # Create custom instance with ruby method
+    code = dedent(
+        """
+        $evm.log(:info, "Hello World")
+        """
+    )
+    instance = custom_instance(ruby_code=code)
+
+    # Create dynamic dialog
+    service_dialog = appliance.collections.service_dialogs
```

Comment:
If you are going to use this variable, can you change the name to `service_dialog_collection` or something that says that the variable is a collection? `service_dialog` name is slightly misleading.